persistence/sql/persister_consent.go

```diff
@@ -777,3 +785,20 @@ func (p *Persister) FlushInactiveLoginConsentRequests(ctx context.Context, notAf
 
 	return nil
 }
+
+func (p *Persister) mySQLConfirmLoginSession(ctx context.Context, session *flow.LoginSession) error {
+	err := p.Connection(ctx).Create(session)
+	if err != nil {
```

Comment:
Shouldn't we check for UniqueViolation instead?

Reply:
fixed